egs/ami/s5b/local/nnet3/run_ivector_common.sh

```diff
@@ -64,18 +65,18 @@ if [ $stage -le 2 ]; then
   fi
 
   for datadir in ${train_set}_sp dev eval; do
-    utils/copy_data_dir.sh data/$mic/$datadir data/$mic/${datadir}_hires
+    utils/copy_data_dir.sh data/$mic/$datadir data/$mic/${datadir}_hires${feature_type}
   done
 
   # do volume-perturbation on the training data prior to extracting hires
   # features; this helps make trained nnets more invariant to test data volume.
-  utils/data/perturb_data_dir_volume.sh data/$mic/${train_set}_sp_hires
+  utils/data/perturb_data_dir_volume.sh data/$mic/${train_set}_sp_hires${feature_type}
 
   for datadir in ${train_set}_sp dev eval; do
     steps/make_mfcc.sh --nj $nj --mfcc-config conf/mfcc_hires.conf \
```

Comment:
@GaofengCheng Did you just try it on AMI? AMI is 16kHz setup with 80 hrs training, it may doesn't have enough examples to distinguish classes using higher order features. Did you try 64-dimMFCC  on 300 hrs SWBD data?